ui/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.test.js

```diff
@@ -41,6 +41,8 @@ describe('Confirm Page Container Content', () => {
   });
 
   it('render ConfirmPageContainer component with simulation error', async () => {
+    process.env.EIP_1559_V2 = false;
```

Comment:
Make sure to reset `EIP_1559_V2` to its previous value after the test or else this will bleed over into other tests.

Reply:
false is default value

Reply:
What if you have EIP_1559_V2 set to true locally?